server/internal/app/file_uploader.go

```
@@ -0,0 +1,360 @@
+			if err != nil {
+				return nil, echo.NewHTTPError(http.StatusBadRequest, "Failed to get file chunk")
+			}
+
+			return uploadManager.handleChunkedUpload(ctx, usecases, op, teamID, fileID, chunkNum, totalChunks, f)
+		}),
+	)
+
+}
+
+func (m *UploadManager) handleChunkedUpload(ctx context.Context, usecases *interfaces.Container, op *usecase.Operator, teamId, fileID string, chunkNum, totalChunks int, file multipart.File) (interface{}, error) {
+	defer file.Close()
```

golangci-lint (1.64.8)

Error return value of `file.Close` is not checked

(errcheck)